ci/ray_ci/automation/docker_tags_lib.py

```diff
@@ -97,6 +99,38 @@ def list_image_tag_suffixes(
     return tag_suffixes
 
 
+def pull_image(image_name: str) -> None:
+    """
+    Pull image from Docker Hub.
+    """
```

Comment:
- this also can be pulling image from other sources.
- change this to `_pull_image`?
- and drop the doc comment; don't comment things that are obvious. like this is used 3 lines to comment 2 lines of code (that is obvious by reading the code).

Reply:
I'm keeping it without the underscore for now since we are calling it from another file